web-ui/src/main/resources/catalog/components/usersearches/partials/portalswitcher.html

```diff
@@ -25,14 +25,14 @@
         {{::info['system/site/name']}}
       </a>
     </li>
-    <li class="divider" data-ng-if="portals.length > 0"></li>
-    <li class="dropdown-header" translate="" data-ng-if="portals.length > 0">
+    <li class="divider" data-ng-if="(portals | filter: {listableInHeaderSelector: true}).length > 0"></li>
```

Comment:
```suggestion
    <li class="divider" data-ng-if="(portals | filter: {listableInHeaderSelector: true}).length > 0  && (nodeId != 'srv')"></li>
```

Don't show the divider in the main portal.